test/test_energy_storage_device.py

```diff
@@ -28,24 +30,31 @@ def test_consistency_constant_evolve_functions(self):
                                dt)
         df.reset()
         P = 15.0  # watts/m2
+        error = 1e-2  # represents leniency of error caused
+                      # by dualfoil, not the code being tested
```

Comment:
would you elaborate on this like you did to explain the precision issue over the initial 20 seconds in the test where you compare dualfoil run vs cap.